### ISSUE TITLE
"prng.cl" file not found #include <prng.cl> 

### ISSUE THREAD

**Issue by mfbenitezp** [dabreegster/uatk-aspics#15]
After running:

`poetry run python gui.py -p model_parameters/default.yml`

I got 

```
<program source>:3:10: fatal error: 'prng.cl' file not found
#include "prng.cl"
```

Apparently it has something with the headers and the way opencl read them in Mac. https://discourse.julialang.org/t/including-header-files-for-opencl-kernels/36551

I need to do other stuff, but later I will try more stuff, but no reasonable explanation about this weird issue.

**Comment by mfbenitezp** [comment]
I have tested the new .env file. the aspics env was created appropriated, but again it stopped in the same error that poetry env.

```
#include "prng.cl"
         ^
```
The full error is 

```
Running a simulation based on model_parameters/default.yml
Loading snapshot from data/processed_data/WestYorkshireSmall/snapshot/cache.npz
Snapshot is 6 MB
Traceback (most recent call last):
  File "gui.py", line 33, in <module>
    main()
  File "/Users/fbenitez/opt/anaconda3/envs/aspics/lib/python3.7/site-packages/click/core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
  File "/Users/fbenitez/opt/anaconda3/envs/aspics/lib/python3.7/site-packages/click/core.py", line 1053, in main
    rv = self.invoke(ctx)
  File "/Users/fbenitez/opt/anaconda3/envs/aspics/lib/python3.7/site-packages/click/core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/fbenitez/opt/anaconda3/envs/aspics/lib/python3.7/site-packages/click/core.py", line 754, in invoke
    return __callback(*args, **kwargs)
  File "gui.py", line 16, in main
    simulator, snapshot, study_area = setup_sim(parameters_file)
  File "/Users/fbenitez/Documents/ResearchATI/EcoTwins_Rust/rampfs/ramp/loader.py", line 59, in setup_sim
    simulator = Simulator(snapshot, parameters_file, gpu=True)
  File "/Users/fbenitez/Documents/ResearchATI/EcoTwins_Rust/rampfs/ramp/simulator.py", line 84, in __init__
    program.build(options=[f"-I {kernel_dir}"])
  File "/Users/fbenitez/opt/anaconda3/envs/aspics/lib/python3.7/site-packages/pyopencl/__init__.py", line 526, in build
    options_bytes=options_bytes, source=self._source)
  File "/Users/fbenitez/opt/anaconda3/envs/aspics/lib/python3.7/site-packages/pyopencl/__init__.py", line 570, in _build_and_catch_errors
    raise err
pyopencl._cl.RuntimeError: clBuildProgram failed: BUILD_PROGRAM_FAILURE - clBuildProgram failed: BUILD_PROGRAM_FAILURE - clBuildProgram failed: BUILD_PROGRAM_FAILURE

Build on <pyopencl.Device 'Intel(R) UHD Graphics 630' on 'Apple' at 0x6000013040c0>:

<program source>:1:10: fatal error: 'prng.cl' file not found
#include "prng.cl"
         ^
===========================================================================
Build on <pyopencl.Device 'AMD Radeon Pro 5500M Compute Engine' on 'Apple' at 0x6000013040e0>:

<program source>:3:10: fatal error: 'prng.cl' file not found
#include "prng.cl"
         ^

(options: -I ramp/kernels -I /Users/fbenitez/opt/anaconda3/envs/aspics/lib/python3.7/site-packages/pyopencl/cl)
(source saved as /var/folders/n8/gpdfd81x2xqg81bv12th7p840000gr/T/tmp7kgfbq9r.cl)
```

**Comment by mfbenitezp** [comment]
I tried again to address this problem by:
1. Initially installing homebrew, then poetry
2. Set poetry env use to the full path where the update version of python was installed.
3. Poetry installs correctly, but the model did not work.
4. I installed Conda, update the dependencies.
5. I installed the lastest OpenCL drives using https://documen.tician.de/pyopencl/misc.html
6. Conda env also did not work. Same issue.
